pages/04.Reference/08.network/03.known-issues/docs.en.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 'Known Issues'
+published: true
+date: '08-14-2019 10:51'
+taxonomy:
+    category:
+        - docs
+---
+
+## Overview
+
+### Idle TCP sessions being closed
+
+**Problem Statement:**\
```

Comment:
Not sure why we have backslash at the end of the line? If it is to force carriage return, one additional carriage return should split heading from paragraph.